libbuteosyncfw/profile/SyncResults.h

```diff
@@ -76,12 +76,15 @@ class SyncResults
      */
     enum MinorCode {
 
-        //NO-Error
+        // No error
         NO_ERROR = 0,
 
-        //These error codes are mapped to syncAgentConsts.h
+        // Following error codes are mapped (and extending)
+        // to SyncAgentConsts.h from buteo-syncml repository.
```

Comment:
Wonder if there's some point in this coupling with buteo-syncml or is it just providing confusion.